dataactvalidator/config/sqlrules/sqlRules.csv

```diff
@@ -58,10 +58,12 @@ B7,GrossOutlaysDeliveredOrdersPaidTotal (FYB or CPE)= USSGL(4902 + 4908 + 4982).
-B11,"Must be valid 3-digit object class as defined in OMB Circular A-11 Section 83.6, or a 4-digit code which includes a 1-digit prefix that distinguishes direct, reimbursable, and allocation obligations. Do not include decimal points when reporting in the Schema.","Failed rule: Must be valid 3-digit object class as defined in OMB Circular A-11 Section 83.6, or a 4-digit code which includes a 1-digit prefix that distinguishes direct, reimbursable, and allocation obligations. Do not include decimal points when reporting in the Schema.",n,program_activity,warning,,b11_object_class_program_activity_1
-B11,"Must be valid 3-digit object class as defined in OMB Circular A-11 Section 83.6, or a 4-digit code which includes a 1-digit prefix that distinguishes direct, reimbursable, and allocation obligations. Do not include decimal points when reporting in the Schema.","Failed rule: If object class is 4 digits, check that 1 digit prefix is consistent with value for ByDirectReimbursableFundingSource",n,program_activity,warning,,b11_object_class_program_activity_2
+B11,"Must be valid 3-digit object class as defined in OMB Circular A-11 Section 83.6, or a 4-digit code which includes a 1-digit prefix that distinguishes direct, reimbursable, and allocation obligations. Do not include decimal points when reporting in the Schema.","Failed rule: Must be valid 3-digit object class as defined in OMB Circular A-11 Section 83.6, or a 4-digit code which includes a 1-digit prefix that distinguishes direct, reimbursable, and allocation obligations. Do not include decimal points when reporting in the Schema.",n,award_financial,fatal,,b11_award_financial_1
+B11,"Must be valid 3-digit object class as defined in OMB Circular A-11 Section 83.6, or a 4-digit code which includes a 1-digit prefix that distinguishes direct, reimbursable, and allocation obligations. Do not include decimal points when reporting in the Schema.","Failed rule: If object class is 4 digits, check that 1 digit prefix is consistent with value for ByDirectReimbursableFundingSource",n,award_financial,fatal,,b11_award_financial_2
+B11,"Must be valid 3-digit object class as defined in OMB Circular A-11 Section 83.6, or a 4-digit code which includes a 1-digit prefix that distinguishes direct, reimbursable, and allocation obligations. Do not include decimal points when reporting in the Schema.","Failed rule: Must be valid 3-digit object class as defined in OMB Circular A-11 Section 83.6, or a 4-digit code which includes a 1-digit prefix that distinguishes direct, reimbursable, and allocation obligations. Do not include decimal points when reporting in the Schema.",n,award_financial,warning,,b11_award_financial_3

```

Comment:
It looks like the feedback we're giving a user for `b11_award_financial_1` is the same as we're giving them for `b11_award_financial_3`. If that's the case, would it make sense to combine the SQL?
```sql
AND (af.object_class NOT IN (SELECT object_class_code FROM object_class)
     OR af.object_class = '000')
```